index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js"></script>
+
+  <title>
+      San's Portfolio
+  </title>
+</head>
+<body>
+
+    <section id="home">
+    <header>
+      <a href="#" class="logo">SAN ROBINSON</a>
+      <ul id="menu"> 
+          <li><a href="#home" class="active">home</a></li>
+          <li><a href="#about">about me</a></li>
+          <li><a href="#portfolio">porfolio</a></li>
+          <li><a href="SanRSWEResume.html">resume</a></li>
+      </ul>
+    </header>
+
+      <center><h1><a href="https://github.com/saanish3" id="btn">explore my git</a></h1></center>
+      <br></a>
+      <br></a>
+      <br></a>
+      <center><img src=emojiii-removebg-preview.png alt = "reddit avatar"> </center>
+
+      
+    </section>
+    <section id="about">
+      <div>
```

Comment:
There is nothing wrong with using div but we want to use semantic html when we can. Because you already have a section tag you don't need this div tag. You can just have 
```
<section>
<h1>
<p>
....
```